django/forms/widgets.py

```diff
@@ -875,10 +875,10 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
```

Comment:
Does `attrs` still have a use here? I can think of a number of problems with this.

Do `attrs` still apply to a wrapping element? (Or could they if the template was overridden?)
Either way, the documentation should be updated to state that they are ignored or to be used for that purpose.

I also think that this could break existing code that uses `attrs`. Perhaps if `date_attrs` or `time_attrs` is `None` they should fall back to using `attrs` for backwards compatibility?

Otherwise, if `attrs` no longer has a use here, we should issue a deprecation warning and, in future, raise an exception if `attrs` is used for this widget?